content/docs/reference/design-docs/train-sim-v3/driver-behavior-modules.en.md

```diff
@@ -138,6 +138,17 @@ To understand how this algorithm is designed, we need to consider two example ca
 - For step 1 and 2: if a neutral zone and a breaking instruction overlap, both are most constraining to different state properties: the neutral zone affects pantograph state, and the breaking instruction affects speed. The final state has to be a combination of both.
 - For step 3: We need to truncate integration steps to avoid overshoots, and thus avoid the neeed for feedback loops. Idealy, we want to truncate to the exact overshoot location. This overshoot location is not the same as the initial `dt` for the overshot constraint.
 
+#### Why do we need to reconcile candidates?
+
+Here is an example of a numerical integration with two driving instructions (in red) resulting in two candidate curves (in solid white). The first driving instruction represents a braking curve, the second one is a speed limit.
```

Comment:
The candidate curves are black on my machine

Reply:
Ah, of course, just had the dark theme enabled in the drawing app